whylog/tests/tests_log_reader/test_files/001_most_basic/investigation_results.txt

```diff
@@ -0,0 +1,3 @@
+-
```

Comment:
Consider investigation_results files as yaml files. Then _investigation_form_file will be simpler without ugly eval invocation.